src/agents.erl

```diff
@@ -28,14 +30,17 @@
 -callback update_agent(position(), agent_state(), environment(), config()) ->
     environment().
 
--callback partition(environment(),
-                    Colours :: pos_integer(),
-                    Parts :: pos_integer()) ->
-    [[{tile(), agents()}]].
-
 -callback get_positions(agents(), tile()) ->
     [position()].
 
+-callback get_list(agents()) ->
+    [agent()].
+
+-callback get_tiles(pos_integer(), environment()) ->
+    [{dimension(), tile()}].
+
+-callback update_tiles({tile(), environment()}, environment(), config()) ->  %% ???????????? tile() ??
```

Comment:
Line 42 is too long: -callback update_tiles({tile(), environment()}, environment(), config()) ->  %% ???????????? tile() ??.